### PR TITLE
Chore/add-spec-coverage

### PR DESCRIPTION
**Pull request by renatamarques97**
There is some broken and missing specs.
Simplecov gem simplifies the process of recognizing of what is been tested and what is missing.
So, I added simplecov to the project.

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -7,3 +7,8 @@ end
 APP_PATH = File.expand_path('../config/application', __dir__)
 require_relative '../config/boot'
 require 'rails/commands'
+if ENV['RAILS_ENV'] == 'test'
```
Comment (marclerodrigues) [inline bin/rails]: Move this to the `spec/rails_helper.rb`, please. This way we can skip the env check.

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_10_20_162009) do
+ActiveRecord::Schema.define(version: 2019_10_20_164632) do
```
Comment (marclerodrigues) [inline db/schema.rb]: Why this change as necessary?

File: bin/rails (hunk 1 above)
Comment (renatamarques97) [reply bin/rails]: alright

File: db/schema.rb (hunk 2 above)
Comment (renatamarques97) [reply db/schema.rb]: I don't know if its really necessary, this version changed when I made the setup (rails db:migrate)
